build/configs/common_download.sh

```diff
@@ -258,7 +258,7 @@ download_specific_partition()
 		echo "No corresponding binary for the partition ${parts[$partidx]}"
 		echo "Download $exe_name FAILED!"
 	else
-		board_download $TTYDEV ${offsets[$partidx]} ${exe_name} ${sizes[partidx]} ${parts[$partidx]}
+		board_download $TTYDEV ${offsets[$partidx]} ${exe_name} ${sizes[partidx]} ${parts[$partidx]} $TARGET
```

Comment:
As you know, this file is common for entire chipset. Is this right modification in common? If this is required in RTK chipset only, it should be managed in RTK specific script. Could you confirm this?

Reply:
I checked artik, its working with 6316PR